core/src/main/scala/org/dbpedia/spotlight/db/memory/MemoryStore.scala

```diff
@@ -189,6 +198,11 @@ object MemoryStore {
     s
   }
 
+  def loadVectorStore(in: InputStream) = {
```

Comment:
could you put a return type and just return the `load` statement into one line ?